composer.json

```diff
@@ -110,6 +110,13 @@
         "ext-redis": "If doctrine cache works on top of redis"
     },
     "scripts": {
+        "compile": [
+            "app/console -n doctrine:migrations:status",
```

Comment:
It's an utility to see the current status before the migrations, but could be removed without problems if not wanted.